src/components/app/details/triggerView/workflow/nodes/staticNode.tsx

```diff
@@ -19,12 +19,18 @@ export interface StaticNodeProps {
     regex?: string
     isRegex?: boolean
     primaryBranchAfterRegex?: string
+    handleGoToWorkFlowEditor?: (...args)=>void
 }
 
 export class StaticNode extends Component<StaticNodeProps> {
     renderCardContent() {
         return (
-            <div className="workflow-node workflow-node--static">
+            <div
+                className={`workflow-node workflow-node--static ${
+                    this.props.branch === 'Not Configured' ? 'cursor workflow-node--dash' : ''
```

Comment:
Can we create some constants for `'Not Configured'`

Reply:
created a const in constants file